modules/ROOT/pages/database-administration/standard-databases/errors.adoc

```diff
@@ -256,3 +256,36 @@ You can remove it only by running the xref:reference/procedures.adoc#procedure_d
 The one exception to this rule is for the built-in `system` database.
 Any quarantine for that database is removed automatically after instance restart.
 ====
+
+
+[[overloaded]]
+== Overloaded servers
+
+There can be situations when we would like to relieve an overloaded server to moving some of the databases hosted on it.
+In order to to this, we have limited deallocation commands such as `dbms.cluster.deallocateNumberOfDatabases()` which can
+move some databases off a certain server to other servers which do not have as much load.
+
+.Deallocating three databases from a server
+[source, cypher]
+----
+neo4j@system> CALL dbms.cluster.deallocateNumberOfDatabases("server1", 3);
+----
+
+[NOTE]
+====
+The `dbms.cluster.deallocateNumberOfDatabases()` procedure is only available starting in 5.22.
+====
+
+Alternatively, imagine if we added three new servers and want to reallocate a limited number of databases around the servers to rebalance.
```

Comment:
Same here, I think it'll be better to have all methods for reallocation in one section.